recipe/meta.yaml

```diff
@@ -17,15 +17,15 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.6,<3.10
     - pip
     - setuptools
     - wheel
   run:
-    - python >=3.6
+    - python >=3.6,<3.10
```

Comment:
and here keep the python >=3.6 here.  For testing you can use that pinning, but on building final package via c3i turn testing off.